src/components/sections/Section/Section.scss

```diff
@@ -0,0 +1,9 @@
+.pages__section {
+  width: 100vw;
+  content-visibility: auto;
+  contain-intrinsic-size: 2px 1024px;
```

Comment:
Why these two values? It [seems](https://drafts.csswg.org/css-sizing-4/#:~:text=The%20first%20value%20represents%20the%20contain%2Dintrinsic%2Dwidth%20value) they're the width and height, respectively. Have you gotten them inverted by any chance (1024px is a common width value)? Considering the responsive versions, maybe we can replace the fixed 1024px for something like 90vw/100vw/% or something.

Could we use `auto` as stated [here](https://web.dev/content-visibility/#:~:text=is%20a%20new-,auto%20keyword,-for%20contain%2Dintrinsic) to help with the [last remembered size](https://drafts.csswg.org/css-sizing-4/#last-remembered)?

Maybe we could take these values with the mobile version as the baseline with the width as 90/100vw and the height of 380px, something like `contain-instrinct-size: auto 90vw 380px`.
<img width="390" alt="CleanShot 2022-03-04 at 10 35 49@2x" src="https://user-images.githubusercontent.com/381395/156772997-e3043184-d1a9-4e60-a9b9-d17165d68778.png">

Reply:
I thought about using `auto` too, but after watching this [video](https://www.youtube.com/watch?v=FFA-v-CIxJQ) from the Chrome team I realized that the first number does't really matter. In my tests, using `2px` for the width faster than auto, that's why I've left it in there like this.

Reply:
I see. What about the 1024px height, isn't that too much?

Out of curiosity, I was comparing the changes in this Layers tab on DevTools and noticed the height got too large, maybe because of that, not sure. Or that this height even matters much.
Before|After
-|-
![Screen Shot 2022-03-03 at 19 11 15](https://user-images.githubusercontent.com/381395/156777121-e32feda3-cd42-4b13-bc13-f3373dc79f1e.png)|![Screen Shot 2022-03-03 at 19 10 30](https://user-images.githubusercontent.com/381395/156777106-3b07a036-89d3-4be0-8330-17880b0a9873.png)

I had bookmarked that video to watch later and learn more about containment. 
😆

Reply:
Just began watching the video and I understand now why the 1024px (or any other large value) for the height [doesn't seem to be a problem](https://youtu.be/FFA-v-CIxJQ?t=518)!

Reply:
After watching the video and trying to figure out how it works, I'm coming here 🙈 Seems like it doesn't change much, and I haven't noticed any big issue related to the scroll, so I think it's okay to keep it like that.